integration/test/app/simple_test.go

```diff
@@ -0,0 +1,203 @@
+// +build k8srequired
+
+package app
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+	"github.com/giantswarm/microerror"
+	"golang.org/x/net/context"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/integration/key"
+)
+
+const (
+	giantswarm = "giantswarm"
```

Comment:
Is this for the namespace? Otherwise I don't get it. 😉

Reply:
Yes it's for namespace.

Reply:
OK cool then please have namespace in the constant name or just use the string in the test.

I don't see the benefit in having a constant if the name is the same as the value.

Reply:
ack, will change that as well.